GitExtUtils/GitUI/Theming/BrushContainer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Drawing;
+
+namespace GitExtUtils.GitUI.Theming
+{
+    public class BrushContainer : IDisposable
+    {
+        private readonly bool _isSystemBrush;
+
+        public static BrushContainer ForSystemBrush(Brush brush) =>
+            new BrushContainer(brush, isSystemBrush: true);
```

Comment:
nit pick, IDE0090 (general)
A general conversion is done in master, suggest this is done here to give cleaner merge
Style preferences should be documented...
```suggestion
            new(brush, isSystemBrush: true);
```